BLSQ/iaso
# POLIO-728: make active timeline segment yellow

The lightgreen color used to show active segments wasn't different enough from the dark green

## Self proof reading checklist

## Changes

- Changed the color in styling for active step 
- changed the string sent by backend from lightgreen to yellow


## How to test

Explain how to test your PR.
Check a budget with a incomplete segment. Worst case, create one with just the first step

## Print screen / video

![Screenshot 2022-12-14 at 14 45 23](https://user-images.githubusercontent.com/38907762/207611627-7cb5d24e-af12-4210-86f1-191a04933b4b.png)
